TWLight/resources/views.py

```diff
@@ -60,6 +60,25 @@ def get_context_data(self, **kwargs):
             partner=partner, status=Application.SENT).count()
 
         context['total_apps_approved_or_sent'] = context['total_apps_approved'] + context['total_apps_sent']
+        
+        # This if else block supports the template with the number of accounts available 
+        partner_streams = Stream.objects.filter(partner=partner)
+        if partner_streams.count() > 0:
+            context['total_accounts_available_current'] = {}
+            for stream in partner_streams:
+                context['total_apps_approved_or_sent_stream'] = User.objects.filter(
```

Comment:
This is a little confusing. It's a context per stream, but because it's in a for loop all that's passed to the template is the count for the last stream. It should broadly work as intended, but is a little confusing from a code point of view, and actually has a chance of not working as intended if there are no approved or sent applications for that last stream. 

Instead, consider making this a normal variable and turning partner_streams (Line 148) into a context variable to use as a check in the template.